kubernetes-client/src/main/java/io/fabric8/kubernetes/client/PropagationPolicyConfigurable.java

```diff
@@ -17,5 +17,17 @@
 
 public interface PropagationPolicyConfigurable<T>
 {
+  /**
+   * Whether and how garbage collection will be performed.
+   * Either this field or OrphanDependents may be set, but not both.
+   * The default policy is decided by the existing finalizer set in the metadata.finalizers and the resource-specific default policy.
+   * Acceptable values are:
+   *    'Orphan' - orphan the dependents;
+   *    'Background' - allow the garbage collector to delete the dependents in the background;
+   *    'Foreground' - a cascading policy that deletes all dependents in the foreground.
+   *
+   * @param propagationPolicy propagation policy in form of string
+   * @return resource
+   */
```

Comment:
This JavaDoc will be scrambled when processed (no line breaks), try any of the proposed alternatives.
```suggestion
  /**
   * Whether and how garbage collection will be performed.
   * Either this field or OrphanDependents may be set, but not both.
   *
   * <p>The default policy is decided by the existing finalizer set in the metadata.finalizers and the resource-specific default policy.
   * 
   * <p>Acceptable values are:
   * <ul>
   * <li>'Orphan' - orphan the dependents;</li>
   * <li>'Background' - allow the garbage collector to delete the dependents in the background;</li>
   * <li>'Foreground' - a cascading policy that deletes all dependents in the foreground.</li>
   * </ul>
   *
   * @param propagationPolicy propagation policy in form of string
   * @return resource
   */
```
Or:
```suggestion
  /**
   * Whether and how garbage collection will be performed.
   * Either this field or OrphanDependents may be set, but not both.
   *
   * <p>The default policy is decided by the existing finalizer set in the metadata.finalizers and the resource-specific default policy.
   *
   * <p>Acceptable values are:
   * <br>Orphan' - orphan the dependents;</li>
   * <br>Background' - allow the garbage collector to delete the dependents in the background;</li>
   * <br>Foreground' - a cascading policy that deletes all dependents in the foreground.</li>
   *
   * @param propagationPolicy propagation policy in form of string
   * @return resource
   */
```